runtime/runtime.m

```diff
@@ -2901,4 +2901,4 @@ -(int) getHandle
 }
 @end
 
-#pragma clang diagnostic pop
\ No newline at end of file
+#pragma clang diagnostic pop
```

Comment:
^ check your editor settings, it seems you change the last line endings very frequently